Kajabi/sage-lib
# [SAGE-765] - fix(sage-react): input was using an invalid prop 'inputType'

Prop should have been 'type'

## Description
It was called out that the React component `Input` was raising a warning about an invalid prop name `inputType`. 

## Screenshots

|  Before  |  After  |
|--------|--------|
|<img width="1595" alt="image" src="https://user-images.githubusercontent.com/1633290/181101523-29975afb-3a15-42a9-a623-79546ac27c8e.png">||


## Testing in `sage-lib`
1. Navigate to Storybook `input-email`...[here](http://localhost:4100/?path=/docs/sage-input--input-email)
2. Enter an invalid value and you should be greeted with a tooltip similar to the one below. 
<img width="321" alt="image" src="https://user-images.githubusercontent.com/1633290/181371113-a2dd4d0e-38f1-46e4-8444-41855cf9b5f2.png">
3. Should not see any warnings related to Hero or `suppressDefaultClass` in the console.
